actors/builtin/miner/miner_actor.go

```diff
@@ -894,7 +897,10 @@ func (a Actor) verifyWindowedPost(rt Runtime, st *State, onChainInfo *abi.OnChai
 	postRandomness := crypto.DeriveRandWithMinerAddr(crypto.DomainSeparationTag_WindowedPoStChallengeSeed, randomnessK, rt.Message().Receiver())
 
 	store := adt.AsStore(rt)
-	provingSet := adt.AsArray(store, st.ProvingSet)
+	provingSet, err := st.ComputeProvingSet(store)
```

Comment:
Now that windowed post is in, we don't need this method (it used to persist the proving set).

Just check for faults at line 909 in the existing iteration over the proving set. Delete `ComputeProvingSet`.

Reply:
Noted. derp.

I decided to move this into there nonetheless to keep verifySmaller and given that the compute function (substracting faults from the set) will be used outside of actors, so could be useful.

Happy to revert. What do you think of my above comment?